x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/integration_tests/execute_action.test.tsx

```diff
@@ -78,7 +79,7 @@ describe('When using execute action from response actions console', () => {
     enterConsoleCommand(renderResult, 'execute --command="ls -al"');
 
     expect(renderResult.getByTestId('test-validationError-message').textContent).toEqual(
-      'The current version of the Agent does not support this feature. Upgrade your Agent through Fleet to use this feature and new response actions such as killing and suspending processes.'
+      'The current version of the Endpoint Agent does not support execute. Upgrade your Elastic Agent through Fleet to the latest version to enable this response action.'
```

Comment:
(optional) we probably should use the `const` with the message in these tests (if possible) to avoid these types of tests failing when we change them.

Reply:
updated 51306ef67bcdea22dbb536c07cdbed06a2dbd35d